canonical-web-and-design/maas-ui
# Fix styling issues from Vanilla class disordering.

## Done
- Fixed styling issues occurring because of Vanilla classes being included in root *and* in the individual apps. This meant that Vanilla classes were out of their usual order, so some styles were being overwritten when they shouldn't have and vice versa. This PR should address all those issues, because now only placeholders and base classes are being included in the apps (except for u-baseline-grid)
- Rewrote a few classes so they no longer include entire mixins or extend classes.
- Replaced custom legacy `.p-cta` classes with Vanilla `.p-contextual-menu`. I think the component was built when Vanilla didn't have icons in buttons.
- Removed `.p-table-menu` from legacy - it was only used in the old machine list which has now been replaced. It's not used in the machine list seen in the KVM details pages.
- Updated the root loading template to look more like the react loading template.

## QA
- Check that the action menu in legacy and react pages look good.
- Check that table sortable in legacy looks good (e.g. go to device list)
- Check that machine details > events shows information icons correctly
- Refresh and navigate between apps and check that the loading template doesn't jump around while the app is loading.

## Fixes
Fixes #1172 
Fixes #1174
